plugins/modules/ntnx_pc_backup_target_v2.py

```diff
@@ -0,0 +1,444 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2021, Prem Karat
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+module: ntnx_pc_backup_target_v2
+short_description: Create, Update and Delete a cluster or object store as the backup target.
+version_added: 2.1.0
+description:
+    - Create, Update and Delete a cluster or object store as the backup target.
+    - For a given Prism Central, there can be up to 3 clusters as backup targets and 1 object store as backup target.
+options:
+    state:
+        description:
+            - State of the backup target whether to create, update or delete.
+            - If C(state) is present, it will create or update the backup target.
+            - If C(state) is set to C(present) and ext_id is not provided then it will create a new backup target.
+            - If C(state) is set to C(present) and ext_id is provided then it will update the backup target.
+            - If C(state) is absent, it will delete the backup target.
+        choices: ['present', 'absent']
+        type: str
+    wait:
+        description:
+            - Wait for the task to complete.
+        type: bool
+        required: false
+    domain_manager_ext_id:
+        description: A unique identifier for the domain manager.
+        type: str
+        required: true
+    ext_id:
+        description: A unique identifier for the backup target.
+        type: str
+        required: false
+    location:
+        description:
+            - Location of the backup target.
+            - For example, a cluster or an object store endpoint, such as AWS s3.
+        type: dict
+        required: false
+        suboptions:
+            cluster_location:
+                description: Location of the cluster.
+                type: dict
+                suboptions:
+                    config:
+                        description: Configuration of the Cluster reference of the remote cluster to be connected.
```

Comment:
This module will not connect anything to PC right ? Does this cluster needs to be a backing up PC registered ? Please mention in detail.

Reply:
If we can only take backup in  cluster which should be registered to the PC, please mention that here.